heft-plugins/heft-storybook-plugin/src/StorybookRunner.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { Terminal } from '@rushstack/node-core-library';
+
+import { IScopedLogger } from '@rushstack/heft';
+import {
+  ISubprocessRunnerBaseConfiguration,
+  SubprocessRunnerBase
+} from '@rushstack/heft/lib/utilities/subprocess/SubprocessRunnerBase';
+
+export interface IStorybookRunnerConfiguration extends ISubprocessRunnerBaseConfiguration {
+  resolvedStartupModulePath: string;
+}
+
+// TODO: Why must this be a different name from the logger in StorybookPlugin.ts?
+const TASK_NAME: string = 'heft-storybook-runnner';
+
+export class StorybookRunner extends SubprocessRunnerBase<IStorybookRunnerConfiguration> {
+  private _logger!: IScopedLogger;
+  private _terminal!: Terminal;
```

Comment:
Hmm are these not already part of the base class? Can we just initialize them in in the constructor?

Reply:
There is no base class.  TypeScript's `TS2564` basically ruins object-oriented programming, because it insists on a kind of correctness that the type system is too weak to prove. It's tempting to disable it.